src/style.css

```diff
@@ -0,0 +1,114 @@
+* {
+  box-sizing: inherit;
+  padding: 0;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: bold;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  box-sizing: border-box;
```

Comment:
```suggestion
```

Reply:
But how do I keep border-box for every element then?